src/Activity.js

```diff
@@ -1,141 +1,103 @@
-    const average = parseFloat((dayData.reduce((acc, elem) => acc += elem[dataType], 0) / dayData.length).toFixed(1))
+    const dayData = this.activityData.filter(data => data.date === date)
+    const average = parseFloat((dayData.reduce((acc, elem) => {
+      return acc += elem[dataType]}, 0) / dayData.length).toFixed(1))
     const overallAverage = {};
     overallAverage[`${dataType}`] = average
     return overallAverage
   }
 
-  getAllUserAverageForDay(date, userRepo, dataType) { //relevent data is string
+  getAllUserAverageForDay(date, userRepo, dataType) {
     const dayData = this.activityData.filter(data => data.date === date)
-    const average = parseFloat((dayData.reduce((acc, elem) => acc += elem[dataType], 0) / dayData.length).toFixed(1))
+    const average = parseFloat((dayData.reduce((acc, elem) => {
+      return acc += elem[dataType]}, 0) / dayData.length).toFixed(1))
     const overallAverage = {}
     let type = dataType;
     if(dataType === 'numSteps') {
       type = 'steps'
     }
-    overallAverage[`${type}`] = average // this is a cleaner object than just the value being returned
+    overallAverage[`${type}`] = average
     return average;
-    // const selectedDayData = this.activityData.filter(data => data.date === date); // userRepo is actual repo
-    // return parseFloat((selectedDayData.reduce((acc, elem) => acc += elem[dataType], 0) / selectedDayData.length).toFixed(1));
   }
 
-  getUserWeekData(id, date, dataType) {
-    let userActivities = this.activityData.filter(data => data.userID === id); //array of only users activities
-    const firstIndex = userActivities.findIndex(x => x.date === date); // index of that date's object
-    const weekData =  userActivities.slice(firstIndex - 6, firstIndex + 1);
-    return weekData.reduce((weekList, day) => { //not being used yet
-      let type = dataType;
-      if(dataType === 'numSteps') {
-        type = 'steps'
-      }
-      let dayData = {date: day.date, amount: day[dataType], unit: type}
-      weekList.unshift(dayData);
-      return weekList;
-    }, [])
-    // should return object with date, unit and unit type
-  }
-
-// replace userData with getUserData
-  userDataForWeek(id, date, userRepo, dataType) { //return steps over a week for specific user on specific date
-    let userActivities = this.activityData.filter(data => data.userID === id); //array of only users activities
-    const firstIndex = userActivities.findIndex(x => x.date === date); // index of that date's object
-    const weekData =  userActivities.slice(firstIndex - 6, firstIndex + 1);
-    return weekData.reduce((week, day) => {
-      let string = `${day.date}: ${day[dataType]}`
-      week.unshift(string)
-      return week
-    }, [])
-
-  }
-
-
-  getStreak(userRepo, id, dataType) { //return 3-day increasing streak for a users step count or active minutes
+  getStreak(userRepo, id, dataType) {
```

Comment:
I think we still need to find a place to use this, im thinking a button somewhere so there is more UI interaction